deepcopy_test.go

```diff
@@ -27,41 +27,35 @@ func ExampleAnything() {
 
 	for _, expected := range tests {
 		actual := MustAnything(expected)
-		fmt.Println(actual)
-	}
-	// Output:
-	// "Now cut that out!"
-	// 39
-	// true
-	// false
-	// 2.14
-	// [Phil Harris Rochester van Jones Mary Livingstone Dennis Day]
-	// [Jell-O Grape-Nuts]
+		if !DeepEqual(expected, actual) {
+			t.Errorf("want '%v', got '%v'", expected, actual)
+		}
+	}
 }
 
 type Foo struct {
 	Foo *Foo
 	Bar int
 }
 
-func ExampleMap() {
+func TestExampleMap(t *testing.T) {
```

Comment:
Since this example doesn't actually show up in the docs (I got rid of copying map as a public interface), we can rename this one, just take out "Example"

(Make it TestMap)